### DISCUSSION TITLE
Unable to access github classroom

### DISCUSSION THREAD

**Discussion by thunder-007** [DiveHQ/backend-internship-task#20] [Q&A]
Unable to access github classroom
![Screenshot from 2023-06-15 15-59-12](https://github.com/DiveHQ/backend-internship-task/assets/75431678/2f02967d-cbbd-48a8-94e0-4f348f2e8eba)
After clicking on the dive button I get this below page with this url https://education.github.com/globalcampus/[object%20Object],[object%20Object]
![Screenshot from 2023-06-15 15-59-24](https://github.com/DiveHQ/backend-internship-task/assets/75431678/437be122-a3cd-48f8-9f71-252d314c0865)

**Comment by jaiakash** [comment]
You can check your profile to access the repo that is created by github classroom. Or click on the link at Octernship page.
Check the PR of that repo to access the github classroom branch.

**Comment by jaiakash** [comment]
My bad, the bug is different. You cant even apply due to his bug.
I have created issue on Github Discussion, https://github.com/orgs/community/discussions/58200
Maybe upvote it so that it can be fixed,

**Comment by prayashm** [comment]
@Thunder-007 can you check now, I see that the capacity has been increased.

**Reply by prayashm** [reply]
I see your repo, but no code in that though.

**Comment by prayashm** [comment]
@ashwinexe Can you help here?

**Comment by imsandeepk** [comment]
We Just have to push the code to private repo that is been created by GitHub classroom, no PR or something right?

**Reply by ashwinexe** [reply]
Please read the **Task submission** section of the Octernships project. It asks to submit using the [GitHub Flow](https://docs.github.com/en/get-started/quickstart/github-flow#following-github-flow)
